packages/pg-meta/src/sql/functions.ts

```diff
@@ -80,15 +80,16 @@ from
         'mode', t2.mode,
         'name', name,
         'type_id', type_id,
-        'has_default', has_default
+        -- Cast null into false boolean
+        'has_default', COALESCE(has_default, false)
```

Comment:
Is there ever a case where `has_default` is `null`?

Reply:
Would've expected there to be none according to this https://github.com/supabase/supabase/blob/d78768cec464f420b765290208fd60ed98b1085d/packages/pg-meta/src/sql/functions.ts#L20

Reply:
Yup some of them get null values without that.

<img width="1457" alt="Screenshot 2025-02-11 at 19 32 06" src="https://github.com/user-attachments/assets/202150b6-3f5f-401f-8fa5-9d6e509bd0c8" />